lib/setupmanagers/qemuhck/qemuhck.rb

```diff
@@ -9,10 +9,10 @@ class QemuHCK
 
     include Helper
 
-    attr_reader :kit, :project
+    attr_reader :kit, :project, :clients_vm
```

Comment:
mayme we cab find better way to rich QemuMachine class or rethink post start commands implementation

Reply:
we don't need this anymore